spec/cypress/support/step_definitions/factory-bot.js

```diff
@@ -0,0 +1,29 @@
+import { Given } from "cypress-cucumber-preprocessor/steps";
+
+const parseArgs = (argsString) => {
+  const args = {};
+  argsString.split(/ and |, /).forEach((argString) => {
+    const [, key, value] = /([^ ]+) "([^"]+)"/.exec(argString);
+    args[key] = value;
+  });
+
+  return args;
+};
+
+expect(parseArgs('start_year "2021"')).to.deep.equal({ start_year: "2021" });
+expect(parseArgs('a "a b" and b "b c"')).to.deep.equal({ a: "a b", b: "b c" });
+expect(parseArgs('a "a b", b "b" and c "d"')).to.deep.equal({
+  a: "a b",
+  b: "b",
+  c: "d",
+});
```

Comment:
Hehe, gotta love sanity checks

Reply:
just a couple unit tests in our end to end test setup 😎